index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Project1</title>
+  </head>
+  <body class="bg-secondary">
+    <div class="container-fluid">
+      <!-- logo & hamburger menu -->
+      <div class="logo-menubar display-flex1">
+        <img src="./images/logo.svg" alt="logo" />
```

Comment:
- [ ] Instead of using pictures you can use icons from [font-awesome](https://fontawesome.com/start). This is optional but ı highly recommended. It is not hard to using.

Reply:
Okay, thanks for this I will put that I mind in my other projects.